tests/CarrierTest.php

```diff
@@ -107,159 +132,349 @@ public function testRoyalmailClassRealValuesAll()
+                'GB', 123123123123, 0.100
+            )
+        );
     }
 
     /**
      * Test to ensure that only the expected empty array is returned from null
      * and incorrect data from the CalculateMethod class
+     *
+     * @return null
      */
     public function testRoyalmailClassNull()
     {
-        $this->assertEquals($this->emptyArray, $this->carrier->getRates(null, 123123123123, 0.100));
-        $this->assertEquals($this->emptyArray, $this->carrier->getRates(null, null, 0.100));
-        $this->assertEquals($this->emptyArray, $this->carrier->getRates('GB', null, 0.100));
-        $this->assertEquals($this->emptyArray, $this->carrier->getRates('GB', null, null));
-        $this->assertEquals($this->emptyArray, $this->carrier->getRates('GB', 123123123123, null));
-        $this->assertEquals($this->emptyArray, $this->carrier->getRates(null, null, null));
+        $this->assertEquals(
+            $this->_emptyArray,
+            $this->_carrier->getRates(
+                null, 123123123123, 0.100
+            )
+        );
+        $this->assertEquals(
+            $this->_emptyArray,
+            $this->_carrier->getRates(
+                null, null, 0.100
+            )
+        );
+        $this->assertEquals(
+            $this->_emptyArray,
+            $this->_carrier->getRates(
+                'GB', null, 0.100
+            )
+        );
+        $this->assertEquals(
+            $this->_emptyArray,
+            $this->_carrier->getRates(
+                'GB', null, null
+            )
+        );
+        $this->assertEquals(
+            $this->_emptyArray,
+            $this->_carrier->getRates(
+                'GB', 123123123123, null
+            )
+        );
+        $this->assertEquals(
+            $this->_emptyArray,
+            $this->_carrier->getRates(
+                null, null, null
+            )
+        );
     }
 
     /**
-     * Test co compare the meta names vs the clean name to check that the correct value
-     * exists and is being used.
+     * Test co compare the meta names vs the clean name to check that
+     * the correct value exists and is being used.
+     *
+     * @return null
      */
     public function testMethodToMetaVsCleanName()
     {
-        foreach ($this->dataClass->getMappingMethodToMeta() as $array => $data) {
+        foreach ($this->_dataClass->getMappingMethodToMeta()
+            as $array => $data) {
             $methodNotExist = false;
-            foreach ($this->dataClass->getMappingCleanNameToMethod() as $method => $methodData) {
+            foreach ($this->_dataClass->getMappingCleanNameToMethod()
+                as $method => $methodData) {
                 // Check the the names are equal
-                if ($data[self::METHOD_NAME_ROW_META_CSV] == $methodData[self::METHOD_NAME_ROW_CLEANNAME_CSV]) {
+                if ($data[self::METHOD_NAME_ROW_META_CSV] == $methodData[self::METHOD_NAME_ROW_CLEANNAME_CSV]
```

Comment:
No, I didn't. Might have to edit the psr-2 requirement, but I couldn't find somewhere to change it from grumphp directly.